cg/store/models.py

```diff
@@ -837,16 +827,18 @@ def is_external(self):
         return self._is_external
 
     @property
-    def sequencing_qc(self) -> bool:
-        """Return sequencing qc passed or failed."""
-        application = self.application_version.application
-        # Express priority needs to be analyzed at a lower threshold for primary analysis
-        if self.priority == Priority.express:
-            one_half_of_target_reads = application.target_reads / 2
-            return self.reads >= one_half_of_target_reads
-        if self.application_version.application.prep_category == PrepCategory.READY_MADE_LIBRARY:
-            return bool(self.reads)
-        return self.reads > application.expected_reads
+    def archive_location(self) -> str:
```

Comment:
```suggestion
    def get_archive_location(self) -> str:
```

Reply:
this is just black formatting stuff, I will not refactor this in this PR